desktop-styles.css

```diff
@@ -504,3 +504,11 @@ button {
   border-radius: 8px;
   border: none;
 }
+
```

Comment:
- [ ] You didn't add linters for javascript which is ESLint. Kindly add the write set of linters 👍

Reply:
There is already ESLint configuration please